e2e-testing/cypress/functions/GeneralSettingsFunction.js

```diff
@@ -0,0 +1,45 @@
+import GeneralSettingsLocator from "../locators/GeneralSettingsLocator";
+
+class GeneralSettingsFunctions{
```

Comment:
Double check (and correct) the name of this file for the tests to run. :)

Reply:
Sure thing. On it.